src/framework/preferences/css/stylus/utils/Helpers.styl

```diff
@@ -21,3 +21,23 @@ linear-gradient() {
         s('linear-gradient(%s)', content);
     }
 }
+
+
+// Mixin for common styles of the "thumb" (slider control)
+// used by both the jQuery and native HTML sliders
+slider-thumb-common() {
+    height: 1.5em;
+    width: 1.5em;
+    background-color: #fff;
+    border: 1px solid #b3b3b3;
+    border-radius: 2em;
+    box-shadow: 4px 2px 3px rgba(0,0,0,0.3);
```

Comment:
Yes - it would be ideal if this scaled with the font-size, but I don't see it as something that would block this from getting merged.

Ideally we'd use `rem` instead of `em` even - but that's another task all together.